lib/ansible/modules/web_infrastructure/jenkins_job.py

```diff
@@ -158,7 +158,7 @@
 
 LXML_IMP_ERR = None
 try:
-    from lxml import etree as ET
+    import xml.etree.ElementTree as ET
```

Comment:
That's a change in dependencies: lxml should then be removed from the documentation of the module.

Reply:
Indeed, done.